docs/setup/settings.asciidoc

```diff
@@ -429,6 +429,12 @@ to display map tiles in tilemap visualizations. By default,
 override this parameter to use their own Tile Map Service. For example:
 `"https://tiles.elastic.co/v2/default/{z}/{x}/{y}.png?elastic_tile_service_tos=agree&my_app_name=kibana"`
 
+| `migrations.batchSize:`
+ | Defines the number of documents migrated at a time. The higher the value, the faster the Saved Objects migration process will be performed at the cost of higher memory consumption. Should migration fail with the `circuit_breaking_exception` exception, you need to set a smaller `batchSize` value. *Default: `1000`*
```

Comment:
nit: 

```suggestion
 | Defines the number of documents migrated at a time. The higher the value, the faster the Saved Objects migration process will be performed at the cost of higher memory consumption. Should the migration fail with a `circuit_breaking_exception`, you need to set a smaller `batchSize` value. *Default: `1000`*
```

Could also consider linking out to the circuit breaker docs https://www.elastic.co/guide/en/elasticsearch/reference/current/circuit-breaker.html